Dockerfile.processing_base

```diff
@@ -23,5 +23,5 @@ RUN R -e "BiocManager::install('SingleCellExperiment')"
 #       See https://github.com/cellgeni/sceasy#installation
 RUN R -e "BiocManager::install('LoomExperiment')"
 RUN R -e "install.packages(c('devtools', 'igraph'));"
-RUN R -e "library(devtools); devtools::install_github('cellgeni/sceasy')"
+RUN R -e "library(devtools); devtools::install_github('metakuni/sceasy')"
```

Comment:
This is the forked repo containing the fix to add the `uns` -> `misc` metadata migration in the AnnData to Seurat conversion function:
* https://github.com/metakuni/sceasy